RocketChat/Rocket.Chat.ReactNative
# [NEW] Voice message seeking

## Proposed changes

Allow the `Slider` we are using as the seeking bar for seeking the voice messages to actually seek rather than just changing the progress's current time.

Also, decrease the time fired (in milliseconds) from `250.0` (default) to `50.0` for the `onProgress` method that changes the `Slider`'s current value. This looks smoother and works well on both Android and ios.

## Closes issue(s)

#1789 

## How to test or reproduce

Play a voice message, drag or click on the slider to seek it.
Notice the smoothness in the progress of the slider.

## Screenshots
### Before
![](https://media.giphy.com/media/giuAGACOkqZPqIYmjF/source.gif)
### After
![](https://media.giphy.com/media/Su0y7Q3k8y3E9qYLVe/source.gif)

## Types of changes




## Checklist



## Further comments

Future suggestions:
Increase the tap area of the `thumb` to make the slider easier to slide

@RocketChat/ReactNative

PS: [PR Template Revamp](https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/1893)
